corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -16,52 +17,72 @@ def translate(t, lang, langs=None):
             return t[lang]
 
 
+def tag_with_brackets(lang):
```

Comment:
are these referenced outside of this template tags file? I think since they aren't template tags themselves and are limited in scope, it might be good to prepend with an underscore so that it's clear these aren't intended to be public methods. Applies to `_tag_with_brackets`, `_tag_with_markup`, `_empty_tag`